test/forms/DescribeYourItemFormProviderSpec.scala

```diff
@@ -54,20 +55,11 @@ class DescribeYourItemFormProviderSpec extends StringFieldBehaviours {
 
     val fieldName = "field2"
     val requiredKey = "describeYourItem.error.field2.required"
-    val lengthKey = "describeYourItem.error.field2.length"
-    val maxLength = 100
 
     behave like fieldThatBindsValidData(
       form,
       fieldName,
-      stringsWithMaxLength(maxLength)
-    )
-
-    behave like fieldWithMaxLength(
-      form,
-      fieldName,
-      maxLength = maxLength,
-      lengthError = FormError(fieldName, lengthKey, Seq(maxLength))
```

Comment:
Did we mean to delete these?

Reply:
`DescribeYourItemFormProvider . field2` does not have any requirement on the length 
So I removed the scenario on the text length

Reply:
Sure